### ISSUE TITLE
[feature] Show instruction dump and incomplete code in the same decompilation

### ISSUE THREAD

**Issue by Trevor159** [skylot/jadx#1400]
title.

It would be nice if jadx had an option to include the whole instruction dump as a comment when generating incomplete code so the users do not have to decompile twice to see the incomplete version and the instruction dump to determine what the different is and fix it.

If it already has this option I cannot find it.

**Comment by jpstotz** [comment]
Please check if you set `Code comments level` in preferences to `DEBUG` if that is what you have in mind.

**Comment by skylot** [comment]
@jpstotz 
As I understand @Trevor159 wants to see fallback dump and inconsistent code at the same time to compare and check code correctness. This can be done, for example, by output dump in comment followed by code, but for long methods it will be hard to compare because of constant scroll back and forth, so I don't think this is a good idea. But in jadx-gui we can show different code in a split view. I was planning to implement that for smali tab, so maybe additional view can have a combo box (or tabs) with modes like 'smali', 'fallback', and maybe others.

**Comment by Trevor159** [comment]
> @jpstotz As I understand @Trevor159 wants to see fallback dump and inconsistent code at the same time to compare and check code correctness. This can be done, for example, by output dump in comment followed by code, but for long methods it will be hard to compare because of constant scroll back and forth, so I don't think this is a good idea. But in jadx-gui we can show different code in a split view. I was planning to implement that for smali tab, so maybe additional view can have a combo box (or tabs) with modes like 'smali', 'fallback', and maybe others.

This is correct about what I am asking. While I personally still think commenting the fallback dump with the code can be useful for people who do not want to use the gui, I understand your reservations and agree that having a split view in the gui would be the best ux for this.
